.travis.yml

```diff
@@ -1,17 +1,17 @@
-language: python
-
-python:
-  - 2.7
-
-install:
-  pip install -r requirements.txt
-  pip install flake8
-
-before-script:
-  sudo bash -c 'echo -e "BRAND_CONSOLE=Test\nCOMPANY_NAME_SHORT=Test" > /etc/xensource-inventory'
-  sudo mkdir -p /etc/sysconfig
-  sudo bash -c 'echo -e "NETWORKING=yes\nNETWORKING_IPV6=no\nIPV6_AUTOCONF=no\nHOSTNAME=localhost.localdomain" > /etc/sysconfig/network'
-
-script:
-  flake8 --select W291,W292 .
-  nosetests
+language: python
+
+python:
+  - 2.7

```

Comment:
Please include at least the target version of python3 and consider including the latest (currently 3.11).